Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN chmod a+rwX /files
 RUN set -x \
 #  && apt-get update \
   && apt-get install -y inotify-tools \
-  && wget -O /files/filebot.deb 'https://app.filebot.net/download.php?type=deb&arch=amd64&version=4.7.2' \
+  && wget -O /files/filebot.deb 'https://app.filebot.net/download.php?type=deb&arch=amd64&version=4.7.7' \
```

Comment:
Seems like a low priority. The risk is that the 1 time that Docker hub builds the Dockerfile, the file isn't what we expect. After that, Docker provides the integrity guarantee for the average user.